datacommons/examples/statvar.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Basic examples for StatisticalVariable-based Data Commons API functions."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import sys
+sys.path.append('../')
+sys.path.append('../../')
+import datacommons as dc
+
+
+def main():
+    # Dcid for Santa Clara County.
```

Comment:
can you make the entire example parameterized:

data = [{place: xx, stat_var = xxx, xx}, {},{}]
for d in data:
   print(xx)
   print(dc.get())

Reply:
done